clean-modular-code/python-expressive-code.md

```diff
@@ -131,21 +134,19 @@ in a numeric format in degrees Fahrenheit to Kelvin.
 
 ```python
 
-def fahr_to_kelvin(fahr):
+def convert_fahrenheit_to_kelvin(temperature_fahr):
```

Comment:
@sneakers-the-rat @willingc so here what i LOVE and like to suggest is

* use a verb for function names 

What i love less is -- if we write everything out now you have

`convert_fahrenheit_to_kelvin(temperature_fahrenheit)`

and as you add more of these and typing and such the line lengths get really long. At some point the code becomes longer and more to read. so i'm really torn about some of these suggestions but also open to merging and discussing separately!